docs/cdn.md

```diff
@@ -0,0 +1,62 @@
+Metaphysics can be queried through its CDN endpoint, `metaphysics-[staging|production]-alt.artsy.net`.
```

Comment:
Not for this PR obviously, but should we talk about promoting the name of this endpoint to something more self-describing? `alt` feels a bit ambiguous.

Reply:
Yeah I had a similar thought... `-cdn`? And maybe it's time to finally drop `-production` from this app's hostname.

Reply:
(We should think about tackling this before this work starts to spread to Eigen)